tests/foreman/ui/test_hostgroup.py

```diff
@@ -57,12 +57,14 @@ def test_update_hostgroup(self, new_name):
         with Session(self.browser) as session:
             make_hostgroup(session, name=name)
             self.assertIsNotNone(self.hostgroup.search(name))
-            self.hostgroup.update(name, new_name=new_name)
-            self.assertIsNotNone(self.hostgroup.search(new_name))
+            for new_name in generate_strings_list(len1=4):
+                with self.subTest(new_name):
+                    self.hostgroup.update(name, new_name=new_name)
+                    self.assertIsNotNone(self.hostgroup.search(new_name))
+                    name = new_name  # for next iteration
 
-    @data(*generate_strings_list(len1=256))
```

Comment:
I am little bit disagree with that and especially after your improvement that will make that particular test to be executed very fast (create only one entity and try to input every value).

I am absolutely fine for case when we have only three 'alpha', 'numeric' and 'alphanumeric to use 'alphanumeric', but not there